src/olympia/editors/helpers.py

```diff
@@ -387,15 +387,19 @@ def render_last_review(self, row):
                                row.review_version_num, row.review_date)
 
     def render_authors(self, row):
+        def uurl(id_, uname):
```

Comment:
:)
`uurl` made line 399 below fit on one line to make it more readable.  I can refactor (uurl doesn't strictly need to be a method either) if that's preferable.